core/src/main/java/io/grpc/ServerInterceptors.java

```diff
@@ -61,6 +61,11 @@ public static ServerServiceDefinition interceptForward(ServerServiceDefinition s
     return interceptForward(serviceDef, Arrays.asList(interceptors));
   }
 
+  public static ServerServiceDefinition interceptForward(BindableService bindableService,
```

Comment:
Also, generally we prefer to break lines along the highest semantic level.  In this case, grouping the params together on the same line rather than breaking them apart.  It costs the same number of lines.